### PR TITLE
docs(firestore-bigquery-export): add BigQuery packages' README files

### LINKED ISSUE #262: Create READMEs for npm packages for firestore-bigquery-export
### [REQUIRED] Step 2: Extension name

This feature request is for extension: firestore-bigquery-export

### What feature would you like to see?

The npm packages are currently missing READMEs, which causes the npm page to be empty (e.g. https://www.npmjs.com/package/@firebaseextensions/firestore-bigquery-change-tracker). There should be README files created in the subdirectory for these packages that briefly describe what they do.

### PR DESCRIPTION
**Pull request by russellwheatley**
Fixes #262

`README.md` files added to the following BigQuery extension packages:
- `firestore-bigquery-change-tracker` package  
- `gen-schema-view` script 
- `import` script

### CODE REVIEW

**Comment by Salakar** [comment]
cc @laurenzlong - we weren't sure on the level of detail you wanted in these, we've added a summary readme to each with links to the full guides - is this what you had in mind?

File: firestore-bigquery-export/firestore-bigquery-change-tracker/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+The `firestore-bigquery-change-tracker` package is a dependency for the official Firebase Extension [_Export Collections to BigQuery_](https://github.com/firebase/extensions/tree/master/firestore-bigquery-export), [_schema views script_](https://github.com/firebase/extensions/blob/next/firestore-bigquery-export/guides/GENERATE_SCHEMA_VIEWS.md) & the [_import Firestore documents script_](https://github.com/firebase/extensions/blob/next/firestore-bigquery-export/guides/IMPORT_EXISTING_DOCUMENTS.md).
```
Comment (laurenzlong) [inline firestore-bigquery-export/firestore-bigquery-change-tracker/README.md:1]: should replace `next` with `master` in the URLs

File: firestore-bigquery-export/scripts/gen-schema-view/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+The `fs-bq-schema-views` script is for use with the official Firebase Extension
+[_Export Collections to BigQuery_](https://github.com/firebase/extensions/tree/master/firestore-bigquery-export).
+
+This script creates a BigQuery view based on a provided JSON schema configuration file. It queries the data from the `firestore-bigquery-export` extension changelog table to generate the view.
+
+A guide on how to use the script can be found [here](https://github.com/firebase/extensions/blob/next/firestore-bigquery-export/guides/GENERATE_SCHEMA_VIEWS.md).
```
Comment (laurenzlong) [inline firestore-bigquery-export/scripts/gen-schema-view/README.md:6]: use `master`, not `next`

File: firestore-bigquery-export/scripts/import/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+The `fs-bq-import-collection` script is for use with the official Firebase Extension [_Export Collections to BiqQuery_](https://github.com/firebase/extensions/tree/master/firestore-bigquery-export).
+
+This script reads all existing documents in a specified Firestore Collection, and updates the changelog table used by the `firestore-bigquery-export` extension.
+
+A guide on how to use the script can be found [here](https://github.com/firebase/extensions/blob/next/firestore-bigquery-export/guides/IMPORT_EXISTING_DOCUMENTS.md).
```
Comment (laurenzlong) [inline firestore-bigquery-export/scripts/import/README.md:5]: use `master` not `next`

**Pull request merged**